.github/workflows/docker-tests.yml

```diff
@@ -36,10 +36,15 @@ jobs:
             gradle-packages-${{ runner.os }}-${{ github.job }}
             gradle-packages-${{ runner.os }}
 
+      - name: Show Github ENV variables
+        run: env | grep ^GITHUB
+
       - name: Build with Gradle
         uses: hypertrace/github-actions/gradle@main
         with: 
           args: dockerBuildImages
+        env:
```

Comment:
Why do we need this?

Reply:
This is required so that existing github action work as they were working before. Those need commit hash for image tag